### PR TITLE
Add ability to stamp a barcode anywhere in a PDF, not just page 1

### PR DESCRIPTION
**Pull request by breckenedge**

### CODE REVIEW

File: spec/pdf_stamper_spec.rb [hunk 1]
```diff
@@ -28,4 +28,12 @@
     pdf = PDF::Stamper.new(File.join(File.dirname(__FILE__), 'test_template.pdf'))
     pdf.datamatrix('this field does not exist', 'this is a barcode')
   end
+
+  it 'should stamp any pages in the document' do
+    pdf = PDF::Stamper.new(File.join(File.dirname(__FILE__), 'multipage_fields.pdf'))
+    pdf.datamatrix('APPOINTMENT_DATA', 'Hello, world!')
```
Comment (jdrnetworking) [inline spec/pdf_stamper_spec.rb]: Does this test fail without the code changes in this PR?  If not, I'd yank the test.

Comment (breckenedge) [reply spec/pdf_stamper_spec.rb]: It enables a visual check and acts as documentation.

Comment (breckenedge) [reply spec/pdf_stamper_spec.rb]: Also provides code coverage.

Comment (breckenedge) [reply spec/pdf_stamper_spec.rb]: Test removed!

**Pull request merged**